sdl_android/src/main/java/com/smartdevicelink/security/SdlSecurityBase.java

```diff
@@ -44,7 +44,8 @@ public void resetParams() {
     
     public void handleInitResult(boolean val) {
     	setInitSuccess(val);
-    	session.onSecurityInitialized();
+		if (session != null)
+			session.onSecurityInitialized();

```

Comment:
So really this should return immediately if the session is null as it is orphaned and will not be used.